src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1545,6 +1547,29 @@ public Egg get(PlayerEggThrowEvent event) {
 			}
 		}, EventValues.TIME_NOW);
 
+		// PlayerStopUsingItemEvent
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerStopUsingItemEvent")) {
+			EventValues.registerEventValue(PlayerStopUsingItemEvent.class, Integer.class, new Getter<Integer, PlayerStopUsingItemEvent>() {
+				@Override
+				public Integer get(PlayerStopUsingItemEvent event) {
+					return event.getTicksHeldFor();
+				}
+			}, EventValues.TIME_NOW);
```

Comment:
I don't think that ticks should be allowed like this Integer, it should just be the Timespan event value for the same value.